Application/EdFi.Ods.AdminApi.AdminConsole/Features/Instances/AddInstance.cs

```diff
@@ -40,7 +40,7 @@ public async Task<IResult> Execute(InstanceValidator validator, IAddInstanceComm
     public class AddInstanceRequest : IInstanceRequestModel
     {
         public int TenantId { get; set; }
-
+        public string? TenantName { get; set; }
```

Comment:
Thanks for adding `TenantName`. This is something we missed on a previous ticket.